src/frontend/shortlist/src/components/preferences/__tests__/PreferenceDropdown.test.js

```diff
@@ -0,0 +1,57 @@
+/*
+TODO(All these vue files do not have props that correspond to the API, needs to be added)
```

Comment:
The codes are passing, but I can't find the API. Let me know where?

Reply:
Just had another thought... I can find out their types through console.log ..... Will give that a try

Reply:
So I went back to use console.log(typeof testModelValue, "one"); for oneSelect file, and it gives me type string. I also tried it in all other files, they all give me string.          Then I tried console.log(emittedPayload, "emittedPayload"); =====>>>>> "ModelValue".            console.log(emittedEvent, "emittedEvent"); =====>>>  { 'update:modelValue': [ [ 'ModelValue' ] ] } emittedEvent                       Interestingly, if I put [] as the type for modelValue, it will still pass the tests